sklearn/cross_validation.py

```diff
@@ -332,6 +332,119 @@ def __len__(self):
         return self.n_folds
 
 
+def disjoint_label_folds(y, n_folds=3):
```

Comment:
But `y` means something different there; there it is the target of prediction over which samples are stratified. This variable is much more like `label in `LeaveOneLabelOut`